docs/resources.rst

```diff
@@ -126,6 +126,9 @@ Solidity Tools
 
 * `Universal Mutator <https://github.com/agroce/universalmutator>`_
     A tool for mutation generation, with configurable rules and support for Solidity and Vyper.
+    
```

Comment:
Our tests are complaining about trailing whitespace here - can you please check?